index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Portfolio:A programmer</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <header class="main-header">
+      <div class="mh-logo">
+      <a class="logo">
+        My Logo
+      </a>
```

Comment:
- [ ] Great job implementing this change requested by the previous reviewer. You however missed one small step. You are required to make the logo a link that can be clicked on and help users of your website to navigate to the home page. This can be easily fixed by adding the `href="#"` attribute like this:
```
      <a href="#" class="logo"> My Logo</a>
```

- [ ] Also, as the previous reviewer requested [here](https://github.com/Ch-AhteshamUlHaq/Portfolio-Ch-Ahtesham-Ul-Haq/pull/1#discussion_r916180450), you are required to stick as much as possible to the chosen Figma design. You are currently missing `font-weight: 700` as specified in the Figma design.
**This is how your design currently looks:**
![image](https://user-images.githubusercontent.com/86318284/177857274-2c452c89-cc0e-42a0-95e0-6a478d4d2ef6.png)

**This is the desired Figma design:**
![image](https://user-images.githubusercontent.com/86318284/177857350-b47a9d38-9565-4c0f-9735-6f828dc50e98.png)

Kindly note so that you remove the underline that comes with creating a link, you will have to provide `text-decoration: none` CSS rule to your logo styling.

Reply:
Done

Reply:
You may please work on your linters before I proceed with the review. I can see this is the last review and it is not allowed to approve a pull request with failing linters. 🙏